.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
```

Comment:
Add .vscode and .DS_store...etc to .gitignore file so that these files won't be uploaded to the repo.